lib/pdf.js

```diff
@@ -252,10 +252,10 @@ let PDFJSClass = (function () {
     };
 
 
-    cls.prototype.parsePDFData = function(arrayBuffer) {
+    cls.prototype.parsePDFData = function(arrayBuffer, password) {
         this.pdfDocument = null;
 
-        let parameters = {password: '', data: arrayBuffer};
+        let parameters = {password: password, data: arrayBuffer};
```

Comment:
line 255: cls.prototype.parsePDFData = function(arrayBuffer, password: '') {
line 258: let parameters = {password, data: arrayBuffer};

Reply:
Yes! I'll make the changes